src/main/scala/com/monsanto/arch/cloudformation/model/resource/ApiGateway.scala

```diff
@@ -0,0 +1,260 @@
+package com.monsanto.arch.cloudformation.model.resource
+
+import com.monsanto.arch.cloudformation.model.{ConditionRef, ResourceRef, Token, `Fn::GetAtt`}
+import spray.json.{DefaultJsonProtocol, JsonFormat}
+import DefaultJsonProtocol._
+
+case class `AWS::ApiGateway::Account`(
+                                       name: String,
+                                       CloudWatchRoleArn: Option[String] = None,
```

Comment:
updated in https://github.com/MonsantoCo/cloudformation-template-generator/pull/89/commits/756377475449f58dd9a3e3a2fbcbd3f3e125b409